resallocserver/maint.py

```diff
@@ -59,6 +61,27 @@ def resource_delete(self, resources=None):
                 resources = QResources(session=session)
                 resources.kill(res_id)
 
+    def resource_logs(self, resources=None):
+        hooks_dir = os.path.join(app.config["logdir"], "hooks")
+        paths = []
+        for resource in resources:
+            # If the resource was specified by its name instead of its ID
+            if not resource.isnumeric():
+                with session_scope() as session:
+                    query = Query(Resource)
+                    query = query.with_session(session)
+                    query = query.filter(Resource.name == resource)
+                    resource = query.one().id
```

Comment:
I wanted to make some abstraction for this code block but I wasn't sure where (maybe `QResources.get_by_name(self, name)`?), so I simply put it here. It should be moved somewhere else IMHO.